pkg/controller/secrets_test.go

```diff
@@ -0,0 +1,193 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package controller
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/verrazzano/verrazzano-operator/pkg/constants"
+	. "github.com/verrazzano/verrazzano-operator/pkg/managed"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Test creating and getting a secret
+func TestKubeSecrets_CreateGet(t *testing.T) {
```

Comment:
I think the _ should be removed from the name and use bumpy case (first letter cap for public)

Reply:
Fixed